packages/sitecore-jss-nextjs/src/utils.ts

```diff
@@ -75,3 +76,9 @@ export const getJssEditingSecret = (): string => {
   }
   return secret;
 };
+
+export const resolvePointOfSale = (site: SiteInfo, language: string): string => {
```

Comment:
This should be moved under the `sitecore-jss` package > `/personalize` sub-module.

Even though the logic is simple, it might be good to have a class (`PosResolver`) as @illiakovalenko suggested. Can either do something similar to what we do with `SiteResolver` (class in base package and instance in template) or keep it a static class like we [had originally](https://github.com/Sitecore/jss/blob/master/packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/pos-resolver.ts) and even keep the [middleware config named `getPointOfSale` as we had it](https://github.com/Sitecore/jss/blob/master/packages/create-sitecore-jss/src/templates/nextjs-personalize/src/lib/middleware/plugins/personalize.ts#L55).